spec/models/logstash_queries/watcher_no_results_query_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+
+describe WatcherNoResultsQuery do
+  let(:query_args) do
+    {
+      affiliate_name: 'affiliate_name',
+      time_window: '1w',
+      min_doc_count: 50,
+      query_blocklist: 'foo, bar, baz biz',
```

Comment:
did you mean for baz biz to be 1? or is it baz, biz?

Reply:
Yeah, `baz biz` should be one element. The purpose is to test that the blocklist elements are being separated by commas into an array:
```ruby
"params.query.raw": [
                  "foo",
                  "bar",
                  "baz biz"
                ]
```